### ISSUE TITLE
Help links for accessibility guidelines

### ISSUE THREAD

**Issue by sephyzeonas** [ckeditor/ckeditor4#3895]
## Type of report

Feature request

## Provide description of the new feature
-

## Expected Result

In the editor integrated link to a help page with accessibility guidelines

A help page/chapter with guide lines how to generate accessible content with the RTE.
A control element (buttons or links) integrated in the RTE to open the accessibility help page/chapter directly from the RTE (e.g. in the footer).
If possible the control element may be context sensitive, e.g. if images are inserted, the control element links directly to the accessibility help chapter for images.        
Help page content / accessibility guidelines

Accessibility guide lines, which point out (with examples) how to meet the WCAG criteria for all functions of the RTE, e.g.:
Images: meaningful alternative texts,
Videos: inter alia subtitles for deaf users or an equivalent text alternative, which describes the information conveyed by the video.
For the accessibility criteria please refer to: https://www.w3.org/TR/WCAG21/
For further recommendations to Authoring Tool accessibility guidelines (ATAG) please refer to: https://www.w3.org/TR/ATAG20/

## Actual Result

If content (e.g. images, videos) is generated no information is provided in the editor how to build up accessible content.

## Other Details

CKEditor version: 4

**Comment by Comandeer** [comment]
Have you checked our [Accessibility Checker](https://ckeditor.com/ckeditor-4/accessibility-checker/)? It provides an interactive wizard that points out the most common accessibility issues in the editor's content.

**Comment by Comandeer** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.

**Comment by mgifford** [comment]
I don't think that Accessibility Checker should be promoted.  It is still using QuailJS last I checked and that's depreciated.  We need to upgrade it to axe-core before this plugin can be promoted without security concerns.
